test/unit/spriteManipulation.js

```diff
@@ -393,3 +393,68 @@ test('setPropRandom set sprite y properties between 50 and 350', async t => {
 
   nativeAPI.reset();
 });
+
+test('prev, next, and rand dance move will throw when not enough dance moves', async t => {
+
+  const subTest = async ({ moveCount = 1, testCode }) => {
+    const nativeAPI = await helpers.createDanceAPI();
+    nativeAPI.play({
+      bpm: 120,
+    });
+
+    // Mock cat animation poses
+    for(let i = 0; i < moveCount; i++) {
+      nativeAPI.setAnimationSpriteSheet("CAT", i, {}, () => {});
+      nativeAPI.world.MOVE_NAMES.push({
+        name: `move${i}`,
+        rest: i == 0,
+      })
+    }
+    nativeAPI.world.fullLengthMoveCount = moveCount;
+    nativeAPI.world.restMoveCount = 1;
+
+    const sprite = nativeAPI.makeNewDanceSprite("CAT", null, {x: 200, y: 200});
```

Comment:
Tiny style nit: why `{ moveCount = 1, testCode }` elsewhere, but no spaces inside the braces on `{x: 200, y: 200}`?

Would be good to get eslint running on the test folder as well...

Reply:
Split into new PR: https://github.com/code-dot-org/dance-party/pull/134